dbt/adapters/databricks/relation_configs/tblproperties.py

```diff
@@ -74,11 +74,19 @@ def from_relation_results(cls, results: RelationResults) -> TblPropertiesConfig:
 
     @classmethod
     def from_relation_config(cls, relation_config: RelationConfig) -> TblPropertiesConfig:
```

Comment:
This file handles iceberg settings as part of tblproperties for incremental tables.  It could maybe be a separate component, but since iceberg support is accomplished via tblproperties, including here.